downstream/modules/eda/proc-eda-set-up-new-project.adoc

```diff
@@ -0,0 +1,36 @@
+[id="proc-eda-set-up-new-project"]
+
+= Setting up a new project
+
+[role="_abstract"]
+
+Projects are a logical collection of rulebooks.
+
+== Prerequisites
+
+* You are logged in to the {EDAcontroller} Dashboard as a Content Consumer.
+* You have set up a credential, if necessary. 
+For more information, refer to the link:https://docs.ansible.com/automation-controller/latest/html/userguide/credentials.html[Credentials]
+section of the {ControllerNameStart} documentation.
+* You have an existing repository containing rulebooks that are integrated with playbooks contained in a repository to be used by Automation Controller.
+
+== Procedure
+
+. Log in to the {EDAcontroller} Dashboard.
+. From the navigation bar, select btn:[Projects] → btn:[Create project].
+. Insert the following:
```

Comment:
If you switch to a definition list above, suggest the same change here for consistency.